webpack/JobWizard/JobWizard.js

```diff
@@ -17,22 +18,31 @@ export const JobWizard = () => {
   const [advancedValues, setAdvancedValues] = useState({});
   const dispatch = useDispatch();
 
-  const setDefaults = useCallback(response => {
-    const responseJob = response.data;
-    const advancedTemplateValues = {};
-    const advancedInputs = responseJob.advanced_template_inputs;
-    if (advancedInputs) {
-      advancedInputs.forEach(input => {
-        advancedTemplateValues[input.name] = input?.default || '';
-      });
-    }
-    setAdvancedValues(currentAdvancedValues => ({
-      ...currentAdvancedValues,
-      effectiveUserValue: responseJob.effective_user?.value || '',
-      timeoutToKill: responseJob.job_template?.executionTimeoutInterval || '',
-      templateValues: advancedTemplateValues,
-    }));
-  }, []);
+  const setDefaults = useCallback(
+    ({
+      data: {
+        advanced_template_inputs,
+        effective_user,
+        job_template: { executionTimeoutInterval, description_format },
+      },
+    }) => {
+      const advancedTemplateValues = {};
+      const advancedInputs = advanced_template_inputs;
+      if (advancedInputs) {
+        advancedInputs.forEach(input => {
+          advancedTemplateValues[input.name] = input?.default || '';
+        });
+      }
+      setAdvancedValues(currentAdvancedValues => ({
+        ...currentAdvancedValues,
+        effectiveUserValue: effective_user?.value || '',
+        timeoutToKill: executionTimeoutInterval || '',
+        templateValues: advancedTemplateValues,
+        description: description_format || '',
+      }));
+    },
+    []
```

Comment:
```suggestion
({
      data: {
        advanced_template_inputs = [],
        effective_user,
        job_template: { executionTimeoutInterval, description_format },
      },
    }) => {
      const templateValues = {};
      advanced_template_inputs.forEach(input => {
        templateValues[input.name] = input?.default || '';
      });
      setAdvancedValues(currentAdvancedValues => ({
        ...currentAdvancedValues,
        effectiveUserValue: effective_user?.value || '',
        timeoutToKill: executionTimeoutInterval || '',
        templateValues,
        description: description_format || '',
      }));
    },
    []
```

Reply:
`advancedTemplateValues` and `templateValues` are different as there are `advanced_template_inputs` and `template_inputs`

Reply:
I just removed the const declaration from line 30 and used `advanced_template_inputs` directly,

and then renamed `advancedTemplateValues` on line 29 to `templateValues`
so in line 40 you can use it directly as `templateValues,` instead of:
`templateValues: advancedTemplateValues,`

Reply:
`templateValues` is a defined const already that is used in this pr: https://github.com/theforeman/foreman_remote_execution/pull/556
https://github.com/theforeman/foreman_remote_execution/blob/dc45ae420854e8af9cb9bcea6a48d81b0f98b4ed/webpack/JobWizard/JobWizard.js#L19-L40

Reply:
alright, then let's revisit it on one of the final PRs, so it won't cause lot of refactoring